compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -65,8 +65,11 @@ error.attempt.expose.non.public.symbol=\
 error.invokable.must.return=\
   this {0} must return a result
   
-error.main.cannot.be.public=\
-  the main function cannot be explicitly marked as public
+error.main.cannot.be.non.public=\
+  the main function cannot be non public
+
+error.invalid.return.with.main=\
+  main function return type cannot be ''{0}''
```

Comment:
In here if we add the fact that only `int` can be returned, fixing the error will be intuitive. WDYT?

Reply:
decided on this since "invalid return type ''{0}'', the main function may have no returns or may only return int" felt lengthy, will change this too.